### PR TITLE
Support running tests in the example runner

### PR DESCRIPTION
**Pull request by matt-gadd**
**Type:** feature

The following has been addressed in the PR:

* [ ] There is a related issue
* [x] All code matches the [style guide](https://github.com/dojo/framework/blob/master/STYLE.md)
* [ ] Unit or Functional tests are included in the PR

**Description:**
Adds support for running tests in browser as part of the example runner. An extra Tests tab is added to each widget example, which runs it's own suite. This means you can dev and test in the example runner with a single command.

Needs some updates from webpack-contrib/cli-build-app

### CODE REVIEW

**Comment by dojo-site** [comment]
This pull request has been deployed to:
* dojo.widgets: https://dojowidgets-hvgnwrrfy.now.sh

**Comment by dojo-site** [comment]
This pull request has been deployed to:
* dojo.widgets: https://dojowidgets-804gwktoy.now.sh

**Comment by dojo-site** [comment]
This pull request has been deployed to:
* dojo.widgets: https://dojowidgets-6ot4zuzze.now.sh

**Comment by dojo-site** [comment]
This pull request has been deployed to:
* dojo.widgets: https://dojowidgets-6ot4zuzze.now.sh

**Comment by dojo-site** [comment]
This pull request has been deployed to:
* dojo.widgets: https://dojowidgets-6ot4zuzze.now.sh

**Comment by dojo-site** [comment]
This pull request has been deployed to:
* dojo.widgets: https://dojowidgets-8c7ixciwh.now.sh

File: src/examples/src/Example.m.css [hunk 1]
```diff
@@ -3,3 +3,9 @@
 	overflow-x: hidden;
 	overflow-y: scroll;
 }
+
+.iframe {
+	width: 100%;
+	height: 95%;
```
Comment (tomdye) [inline src/examples/src/Example.m.css]: why 95%?

Comment (matt-gadd) [reply src/examples/src/Example.m.css]: any bigger and it results in scrollbars within scrollbars

**Comment by dojo-site** [comment]
This pull request has been deployed to:
* dojo.widgets: https://dojowidgets-4qmziked3.now.sh

**Pull request merged**